docs/pages/enroll-resources/server-access/openssh/openssh-manual-install.mdx

```diff
@@ -99,9 +99,7 @@ and `spec.hostname` to the name of the node as you would like users to see it in
 
 The `metadata.labels` field labels the SSH Service instance so you can apply RBAC rules to it.
 
-The `metadata.name` field isn't mandatory, but setting it here will save you some work later.
-
-To generate a new universal unique identifier (UUID) suitable for a `node` name, use the `uuidgen`
+The `metadata.name` field isn't mandatory, but if supplied, it must be a universal unique identifier (UUID).  To generate a new UUID suitable for a `node` name, use the `uuidgen`
```

Comment:
```suggestion
The `metadata.name` field isn't mandatory, but if supplied, it must be a universal unique identifier (UUID).  To generate a new UUID suitable for a `node` name, use the `uuidgen` command
```
Otherwise, "the" doesn't refer to anything in "use the `uuidgen`".